content/posts/fpcast.md

```diff
@@ -0,0 +1,478 @@
+---
```

Comment:
Might be better to rename the file to `function-pointer-cast-removal.md` to be a bit more friendly toward search engines.

Reply:
I will go with `function-pointer-cast-emulation.md`. Since to be completely accurate, we didn't really remove function pointer casting, we just handle it in a different way.

Reply:
I also edited the title.